apps/dashboard/tailwind.config.js

```diff
@@ -62,43 +420,120 @@ export default {
       },
       primary: {
         DEFAULT: 'hsl(var(--primary))',
+        dark: 'hsl(var(--primary-dark))',
+        darker: 'hsl(var(--primary-darker))',
+        base: 'hsl(var(--primary-base))',
+        'alpha-24': 'hsl(var(--primary-alpha-24))',
+        'alpha-16': 'hsl(var(--primary-alpha-16))',
+        'alpha-10': 'hsl(var(--primary-alpha-10))',
         foreground: 'hsl(var(--primary-foreground))',
       },
+      bg: {
+        strong: 'hsl(var(--bg-strong))',
+        surface: 'hsl(var(--bg-surface))',
+        sub: 'hsl(var(--bg-sub))',
+        soft: 'hsl(var(--bg-soft))',
+        weak: 'hsl(var(--bg-weak))',
+        white: 'hsl(var(--bg-white))',
+      },
+      stroke: {
+        strong: 'hsl(var(--stroke-strong))',
+        sub: 'hsl(var(--stroke-sub))',
+        soft: 'hsl(var(--stroke-soft))',
+        white: 'hsl(var(--stroke-white))',
+      },
+      faded: {
+        dark: 'hsl(var(--faded-dark))',
+        base: 'hsl(var(--faded-base))',
+        light: 'hsl(var(--faded-light))',
+        lighter: 'hsl(var(--faded-lighter))',
+      },
+      information: {
+        DEFAULT: 'hsl(var(--information))',
+        dark: 'hsl(var(--information-dark))',
+        base: 'hsl(var(--information-base))',
+        light: 'hsl(var(--information-light))',
+        lighter: 'hsl(var(--information-lighter))',
+      },
+      static: {
+        black: 'hsl(var(--static-black))',
+        white: 'hsl(var(--static-white))',
+      },
       accent: {
-        DEFAULT: 'hsl(var(--accent))',
+        DEFAULT: 'hsl(var(--accent))', // DEPRECATED
       },
       destructive: {
-        DEFAULT: 'hsl(var(--destructive))',
-        foreground: 'hsl(var(--destructive-foreground))',
+        DEFAULT: 'hsl(var(--destructive))', // DEPRECATED
+        foreground: 'hsl(var(--destructive-foreground))', // DEPRECATED
       },
       success: {
         DEFAULT: 'hsl(var(--success))',
+        dark: 'hsl(var(--success-dark))',
+        base: 'hsl(var(--success-base))',
+        light: 'hsl(var(--success-light))',
+        lighter: 'hsl(var(--success-lighter))',
       },
       warning: {
         DEFAULT: 'hsl(var(--warning))',
+        dark: 'hsl(var(--warning-dark))',
+        base: 'hsl(var(--warning-base))',
+        light: 'hsl(var(--warning-light))',
+        lighter: 'hsl(var(--warning-lighter))',
+      },
+      away: {
+        dark: 'hsl(var(--away-dark))',
+        base: 'hsl(var(--away-base))',
+        light: 'hsl(var(--away-light))',
+        lighter: 'hsl(var(--away-lighter))',
+      },
+      error: {
+        DEFAULT: 'hsl(var(--error))',
```

Comment:
Please refer to my previous comment about adding too many tokens and variables on the CSS level.

Reply:
I would highly suggest keeping the align UI based approach, as it will ensure on modification of a color token all relevant components will be updated, in-case we would want to change things like color-feature for example. it is mapped perfectly to what naveen is having on Figma and it will simplify our maintenance for new components assell. As copy-paste from align UI snippets will work out of the box.